spec/controllers/dynamic_pages_controller_spec.rb

```diff
@@ -0,0 +1,206 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe DynamicPagesController, type: :controller do
+  fixtures :all
```

Comment:
It might be helpful to make new fixtures for these pages. We could them simplify the tests quite a bit.

Reply:
Will be added in PR 2